playwright-test/atb.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { test, expect, mockAtb } from "./harness";
+
```

Comment:
I don't feel strongly about it, but I know using TypeScript `.ts` files was kind of controversial at DuckDuckGo in the past. This is the first I've seen in the extension. My vote is to just go with it, I don't see the harm. But what do you think @jonathanKingston ?

Reply:
Sam's rationale of the code samples of Playwright are default in TS doesn't to me warrant a departure from the existing codebase standard / direction.

The aim is to move to zero compilation and just because one tool does transpliation for you doesn't really justify it and may confuse new comers on the style.

Reply:
I updated these files to `.js`

Reply:
Cool, thanks :+1: